src/main/java/org/takes/rq/RqFake.java

```diff
@@ -73,7 +73,8 @@ public RqFake(final String method, final String query) {
      * @param query HTTP query
      * @param body HTTP body
      */
-    public RqFake(final String method, final String query, final String body) {
+    public RqFake(final CharSequence method, final CharSequence query,
+                  final CharSequence body) {
```

Comment:
@YamStranger look at other classes and you will see which code style we're using. we indent by 4 spaces always, no exceptions